CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)
 
 ## [Unreleased]
+- `check-ebs-burst-limit.rb`: Fixed period to pass to config from 60 to 120 so as to not have empty values returned sometimes
```

Comment:
This should go under a `### Fixed` header, I will fix it up for you post merge prior to release.

Reply:
In this case I am gonna call it a `### Breaking Changes` and version it as a major so that it does not catch people by surprise.